### PR TITLE
feat(libs): Upgrade @dataesr/react-dsfr version from 0.7.7 to 3.1.1

### PR DESCRIPTION
**Pull request by annelhote**

### CODE REVIEW

File: src/components/Charts/publications/editeurs/dynamique-ouverture/chart-taux-ouverture.js [hunk 1]
```diff
@@ -58,13 +58,13 @@ const Chart = ({ hasFooter, hasComments, id, domain }) => {
...
     Axios.post(ES_API_URL, query, HEADERS).then((response) => {
-      setPublishers(
-        response.data.aggregations.by_publisher.buckets
-          .filter((item) => item.key !== 'Cold Spring Harbor Laboratory')
-          .map((item) => ({ label: item.key, value: item.key })),
-      );
+      const opts = response.data.aggregations.by_publisher.buckets
+        .filter((item) => item.key !== 'Cold Spring Harbor Laboratory')
+        .map((item) => ({ label: item.key, value: item.key }));
+      opts.unshift({ label: capitalize(intl.formatMessage({ id: 'app.all-publishers' })), value: '*' });
```
Comment (poplingue) [inline src/components/Charts/publications/editeurs/dynamique-ouverture/chart-taux-ouverture.js:64]: Maybe we could refactorize in one function this loop with filter() and unshift(), with an explicit name

File: src/components/SimpleSelect/index.js [hunk 2]
```diff
@@ -9,27 +10,19 @@ const SimpleSelect = ({
...
 }) => {
   const selectId = uuidv4();
   return (
     <div className='simple-select'>
       <label htmlFor={selectId}>
-        {label}
-        <select
-          name='select'
-          id={selectId}
+        <div className='simple-select-label'>
```
Comment (poplingue) [inline src/components/SimpleSelect/index.js:18]: No error with `div` inside `label` ?

File: src/components/TodayNumbersSection/index.js [hunk 3]
```diff
@@ -30,7 +30,11 @@ function TodayNumbersSection({ updateDate, title, children }) {
                   {updateDate}
                 </section>
                 <section className='w-100 h-50'>
-                  <Card bodyClassName='bg-soft-blue' href='/'>
+                  <Card
+                    bodyClassName='bg-soft-blue'
+                    className='fr-card--no-border'
```
Comment (poplingue) [inline src/components/TodayNumbersSection/index.js:35]: props `hasBorder={false}` does not work ?

File: src/components/BannerImage/style.scss [hunk 4]
```diff
@@ -43,8 +43,8 @@
 
 .bso-link {
   @include font-size(14, 12);
-  color: var(--blue-dark-125);
   background: #fff;
+  color: var(--blue-dark-125);
```
[the remark this answers is not in the document]
Comment (annelhote) [reply src/components/BannerImage/style.scss:47]: Needed for the button on the banner.

File: src/components/Banner/style.scss [hunk 5]
```diff
@@ -29,9 +31,9 @@
   }
 
   &.sticky {
+    padding: 0;
```
[the remark this answers is not in the document]
Comment (annelhote) [reply src/components/Banner/style.scss:34]: Needed when the banner is sticky on top.

File: src/components/SimpleSelect/index.js (hunk 2 above)
Comment (annelhote) [reply src/components/SimpleSelect/index.js:18]: no problem :)

File: src/components/TodayNumbersSection/index.js (hunk 3 above)
Comment (annelhote) [reply src/components/TodayNumbersSection/index.js:35]: Thank you, I did not know this attr !

**Pull request merged**